### ISSUE TITLE
Problems reading data from Binary store

### ISSUE THREAD

**Issue by bpsourav21** [facebook/react-native#31823]
```
* What went wrong:
Could not determine the dependencies of task ':app:compileDebugJavaWithJavac'.
> Could not resolve all dependencies for configuration ':app:debugCompileClasspath'.
   > Problems reading data from Binary store in C:\Users\sourav\AppData\Local\Temp\gradle7627481342141045842.bin offset 125802 exists? true
 ```
I am getting this error from yesterday and It is barely building my app. I am using `gradle-6.2-all.zip`,
OS: Windows 10
Node version: `12.5.0`
React version: `16.13.1`
React-native version: `0.63.3`
here is my `..\android\build.gradle` file
```
buildscript {
    ext {
        buildToolsVersion = "29.0.3"
        minSdkVersion = 21
        compileSdkVersion = 29
        targetSdkVersion = 29

        googlePlayServicesIidVersion = "16.0.1"
        firebaseMessagingVersion = "21.1.0"
        playServicesVersion = "17.0.0" 
        androidMapsUtilsVersion = "2.2.1"
    }
    repositories {
        google()
        jcenter()
        mavenCentral()
    }
    dependencies {
        classpath("com.android.tools.build:gradle:3.5.4")
        classpath('com.google.gms:google-services:4.3.3')
    }
}

allprojects {
    repositories {
        mavenLocal()
        maven {
            url("$rootDir/../node_modules/react-native/android")
        }
        maven {
            url("$rootDir/../node_modules/jsc-android/dist")
        }

        google()
        jcenter()
        maven { url 'https://www.jitpack.io' }
    }
}
```
Anyone can suggest how can I fix my build issue and what might be the problem here?

_Originally posted by @bpsourav21 in https://github.com/facebook/react-native/issues/31515#issuecomment-875615011_

**Comment by bpsourav21** [comment]
Issue resolved. Closing this issue.

**Comment by SagarVerma123** [comment]
Please provide a solution for this issue.

**Comment by dharshu5** [comment]
Please provide the solution

**Comment by dharshu5** [comment]
@bpsourav21 Please provide the solution

**Comment by dharshu5** [comment]
> Please provide a solution for this issue.

check this I have found the solution
https://stackoverflow.com/questions/68189085/problems-reading-data-from-binary-store-in-tmp-gradle14484220662220949872-bin-o/68770757#68770757

**Comment by bpsourav21** [comment]
Hello, extremely sorry for being late. After some intense debugging I found the reason was for  `react-native-braintree-dropin-ui` this library which I am using on my app. Below is the detailed information about that issue -
https://github.com/wgltony/react-native-braintree-dropin-ui/issues/75

I forked that original project and made some changes for making my app working again. Sharing that fork project below, if anyone needs any help -
https://github.com/bpsourav21/react-native-braintree-dropin-ui
